src/server/pullJobScheduler.ts

```diff
@@ -98,6 +122,66 @@ export const scheduleJob = (pullJob: PullJob) => {
   }
 };
 
+/**
+ * Schedule or re-schedule a custom notification.
+ *
+ * @param custNotification pull job to schedule
+ */
+export const customNotificationScheduleJob = async (
```

Comment:
@ketanreliefapps all related to custom notifications should be in another file, as this one is mostly for pull jobs

Reply:
@AntoineRelief  done